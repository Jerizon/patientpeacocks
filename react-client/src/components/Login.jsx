import React from 'react';

class Login extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
    }
  }


  reqEventful () {
    this.props.eventful();
  }

  reqMeetUP () {
    this.props.meetUp();
  }

  render () {
    return (
      <div>
        <button onClick = {this.reqEventful.bind(this)}>searchEventful</button>
        <button onClick = {this.reqMeetUP.bind(this)}>searchMeetUp</button>
      </div>
    );
  }




}

export default Login;