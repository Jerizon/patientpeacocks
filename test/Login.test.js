import React from 'react';
import ReactDOM from 'react-dom';
import Login from '../react-client/src/components/Login.jsx';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

describe('Component: App', () => {
  it('should match its empty snapshot', () => {
    const tree = renderer.create(
      <Login/>
     ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

describe('Should change redirect to User', () => {
  const redirect = Test.Utils.renderIntoDocument(<Login/>)
});