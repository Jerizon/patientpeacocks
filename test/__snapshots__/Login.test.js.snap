// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: App should match its empty snapshot 1`] = `
<div
  className="loginPage"
>
  <form
    method="POST"
    onSubmit={[Function]}
  >
    <div
      className="login-text"
    >
      Login
    </div>
    <div
      className="loginpg-text"
    >
      Username: 
      <br />
      <input
        type="text"
      />
    </div>
    <div
      className="loginpg-text"
    >
      Password: 
      <br />
      <input
        type="password"
      />
    </div>
    <div
      className="login-button"
    >
      <input
        className="log-button"
        type="submit"
        value="Login!"
      />
    </div>
  </form>
  <div
    className="links"
  >
    <a
      onClick={[Function]}
      style={Object {}}
    >
      Click here to create an account!
    </a>
    <br />
    <a
      onClick={[Function]}
      style={Object {}}
    >
      Back to homepage
    </a>
  </div>
</div>
`;
